### PR TITLE
Move some ndlstm functions to contrib

### PR DESCRIPTION
**Pull request by jvgonzalvo**
Moved `images_to_sequence` and `sequence_to_images` to contrib.layers since the ndlstm module might be removed in the future tensorflow versions. Addresses this issue: https://github.com/tensorflow/tensorflow/issues/16794

### CODE REVIEW

File: tensorflow/contrib/ndlstm/python/lstm1d.py [hunk 1]
```diff
@@ -51,8 +51,8 @@ def ndlstm_base_unrolled(inputs, noutput, scope=None, reverse=False):
   """
   with variable_scope.variable_scope(scope, "SeqLstmUnrolled", [inputs]):
     length, batch_size, _ = _shape(inputs)
-    lstm_cell = rnn_cell.BasicLSTMCell(noutput, state_is_tuple=False)
-    state = array_ops.zeros([batch_size, lstm_cell.state_size])
+    lstm_cell = rnn_cell.BasicLSTMCell(noutput)
+    state = lstm_cell.zero_state(batch_size, inputs.dtype)
```
Comment (ebrevdo) [inline tensorflow/contrib/ndlstm/python/lstm1d.py]: do you need these changes?

**Review by ebrevdo — CHANGES_REQUESTED** [review]

File: tensorflow/contrib/ndlstm/python/lstm1d.py (hunk 1 above)
Comment (selcouthlyBlue) [reply tensorflow/contrib/ndlstm/python/lstm1d.py]: Oh yeah, I accidentally merged the that changed into the master branch of my copy. Removing that now.

Comment (selcouthlyBlue) [reply tensorflow/contrib/ndlstm/python/lstm1d.py]: Reverted those changes. Test please.

**Review by ebrevdo — APPROVED** [review]

**Comment by selcouthlyBlue** [comment]
Test please

**Comment by jhseu** [comment]
Mind fixing `//tensorflow/contrib/layers:layers_test`?

**Comment by selcouthlyBlue** [comment]
Working on that one.

**Comment by selcouthlyBlue** [comment]
Fixed it. Forgot to remove the extra argument that was causing the error. My bad.

**Comment by selcouthlyBlue** [comment]
I checked the Sanity Check logs and some parts of the code I modified seem to exceed the line length limit. Do I push the fixes now or do I wait for the tests to finish?

**Comment by ebrevdo** [comment]
Fix them now and restart tests.

**Comment by selcouthlyBlue** [comment]
Fixed.

**Pull request merged**